.github/workflows/maven.yml

```diff
@@ -3,12 +3,6 @@
 
 name: Java CI with Maven
 
-on:
-  push:
-    branches: [ master ]
```

Comment:
I think we do not need to have this running on all the branches? What would be the benefit of that?

It would be fine to include master (UniTime 4.6) and perhaps maint_UniTime45 (UniTime 4.5, which is our current production version).

Reply:
the benefit of it is that anyone submitting a PR has github run all tests and build process to verify nothing was broken. That is why I'm asking what would be the configuration to have builds run faster but in a way that at least guarantees tests have been run.